lib/openscap_parser/group.rb

```diff
@@ -48,14 +48,6 @@ def parent_id
       @parent_id = parsed_xml.xpath('../@id').to_s
     end
 
-    def parent_type
```

Comment:
I think it needs some explanation. Why is this being deleted? Wouldn't this break stuff?

Reply:
No, we're not using this in compliance-backend at all because we realized that we didn't need it. It ended up just not being necessary

Reply:
I can leave it in though in case we want to use it in the future